redis/src/cluster_routing.rs

```diff
@@ -608,11 +608,18 @@ pub fn is_readonly_cmd(cmd: &[u8]) -> bool {
             | b"BITPOS"
             | b"DBSIZE"
             | b"DUMP"
-            | b"EVALSHA_RO"
+            | b"EVAL" // EVAL on a RO script is a RO operation
```

Comment:
If the user wants to use a readonly eval they can use EVAL_RO, otherwise we will route all EVAL scripts to replicas and won't know if it's a readonly script or not. Lets keep it not readonly and refer users to RO if they want to route it to replicas.

Reply:
Fixed, please, re-review